src/Components/Facility/FacilityHome.tsx

```diff
@@ -222,8 +222,8 @@ export const FacilityHome = (props: any) => {
                 </div>
               </div>
               <div className="flex flex-1 items-center">
-                <div className="mb-6 grid  w-full grid-cols-1 gap-4 md:mb-0 lg:grid-cols-2">
-                  <div className="flex-col justify-between md:flex lg:flex-1 ">
+                <div className="mb-6 grid  w-full   gap-4 sm:grid-cols-2   md:mb-0  lg:grid-cols-2">
+                  <div className="grid flex-col justify-between md:flex lg:flex-1 ">
```

Comment:
You've used `grid` without specifying `grid-cols-{x}`, but with `flex-col`. Why? What does that try to achieve?